src/org/oidc/service/base/enums/ClientAuthenticationMethod.java

```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```

Comment:
Like I mentioned yesterday , lot of the enums would be shared across like this one. Let's keep this in org.oidc.common package.